### ISSUE TITLE
Dropdowns on a topmost window appear behind the window

### ISSUE THREAD

**Issue by skillbert** [cefsharp/CefSharp#657]
Can be reproduced by setting topmost to true in the winforms example.
![image](https://cloud.githubusercontent.com/assets/7593843/5240407/a4a3bee6-7903-11e4-9242-9f24b4b69b01.png)

**Comment by iamarsey** [comment]
Is anybody has any solution to fix this bug? Seem to me that the bug is currently open.

**Comment by amaitland** [comment]
https://code.google.com/p/chromium/issues/detail?id=62037

Needs to be fixed in `Chromium`

**Comment by amaitland** [comment]
https://bitbucket.org/chromiumembedded/cef/issues/1468/dropdowns-are-hidden-behind-browser-window#comment-43607354

**Comment by amaitland** [comment]
`master` has been updated with commit https://github.com/cefsharp/CefSharp/commit/4e50ad8a95a44dc052bdf4014d62dd761bf37215

Changes will be available as part of `71.0.0`

**Comment by amaitland** [comment]
#2576 suggests there is still a problem, reopening.

**Comment by ValeriiSalamatin** [comment]
@amaitland 
Sorry, haven't found it as a duplicate.

Yes, I have tested it on CEF sample application with their new startup parameter. And in this case it works:
![image](https://user-images.githubusercontent.com/44665490/48895843-1e235e80-ee57-11e8-86a6-87cde6dcb94c.png)

**Comment by amaitland** [comment]
I think you'll find this is only an issue with `MultiThreadedMessageLoop = true` (the default), I suspect when `CEF` calls `GetAncestor` it's not correctly returning the `HWND` for the `Form`. 

https://bitbucket.org/chromiumembedded/cef/commits/5969b2bbb83ff22d3276ac3c21e874926e8bb1d8?at=3538#Llibcef/browser/native/browser_platform_delegate_native_win.ccT193

Debugging this is time/cost prohibitive. If someone wishes to they can, a `PR` with changes would be welcome.

As a workaround integrate `CEF` into your existing message loop see https://github.com/cefsharp/CefSharp/wiki/General-Usage#multithreadedmessageloop

Closing as there is a viable workaround.

**Comment by matthid** [comment]
@amaitland Thanks for the pointers, we run into similar issues and I reported https://bitbucket.org/chromiumembedded/cef/issues/2860/dropdowns-are-hidden-behind-browser-window
I tried to analyze why `MultiThreadedMessageLoop` is affected while `false` is not (I tried switching but then it crashes at shutdown with an accessviolation for us), but this thing is incredibly hard to debug.
`MultiThreadedMessageLoop=false` solves the problem only partly (if you are not changing at runtime). Maybe this analysis might ring a bell on where to look further (ideally I'd like to find a viable workaround for now and provide a fix upstream, but given the amount of projects involved it will take quite a while to reach back to us)?

One (almost) working workaround: If we make sure to always start with TopMost and use `MultiThreadedMessageLoop` then it seems to work, but crashes at shutdown. Are there any special requirements for calling shutdown (in any particular thread for example)?
